physionet-django/user/urls.py

```diff
@@ -18,6 +18,9 @@
     path("settings/cloud/aws/", views.edit_cloud_aws, name="edit_cloud_aws"),
     path("settings/orcid/", views.edit_orcid, name="edit_orcid"),
     path("authorcid/", views.auth_orcid, name="auth_orcid"),
+    path("authorcid_login/", views.auth_orcid_login, name="auth_orcid_login"),
```

Comment:
Should these paths only be available if ORCID is enabled, e.g.

```
if settings.ORCID_LOGIN_ENABLED:
    urlpatterns += [
        ...
    ]
```

Reply:
Good idea! Changed